### PR TITLE
Python holistic

### PR DESCRIPTION
**Pull request by rasapala**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -92,3 +92,15 @@ run_python_object_detection:
 	cp build/lib.linux-x86_64-cpython-38/mediapipe/examples/python/ovms_object_detection.py build/lib.linux-x86_64-cpython-38
 	python build/lib.linux-x86_64-cpython-38/ovms_object_detection.py
 
+run_python_holistic_tracking:
+	echo "Running python ovms holistic tracking demo"
+	if [ ! -f video.mp4 ]; then wget -O video.mp4 "https://www.pexels.com/download/video/3044127/?fps=24.0&h=1080&w=1920"; fi
```
Comment (atobiszei) [inline Makefile]: Move video link to variable

File: mediapipe/examples/python/ovms_face_detection.py [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+def grab_frame(cap):
+    success, frame = cap.read()
+    if not success:
+        print("[WARNING] No Input frame")
+        finish()
+        return None
+    return frame
+
+i = 0
```
Comment (atobiszei) [inline mediapipe/examples/python/ovms_face_detection.py]: Rename

File: mediapipe/examples/python/ovms_holistic_tracking.py [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+#
+
+import argparse
+import cv2
+import mediapipe as mp
+import time
+import numpy as np
+import threading
+
+class RequestingThread(threading.Thread):
```
Comment (atobiszei) [inline mediapipe/examples/python/ovms_holistic_tracking.py]: This is basically repeated script for face detection - why we can't reuse code here?

Comment (rasapala) [reply mediapipe/examples/python/ovms_holistic_tracking.py]: I will remove multithreading so it will be only in one example.

**Review by dtrawins — APPROVED** [review]

File: Makefile [hunk 4]
```diff
@@ -56,39 +57,51 @@ run_python_demos_in_docker:
 # Targets to use inside running mediapipe_ovms container
 run_demos: run_holistic_tracking run_face_detection run_iris_tracking run_object_detection run_pose_tracking
 
-run_python_demos: run_python_object_detection
+run_python_demos: run_python_object_detection run_python_holistic_tracking run_python_face_detection
 
 run_object_detection:
-	echo "Running FPS test for object_detection demo"
+	echo "Running FPS test for objectpexels_detection demo"
```
Comment (atobiszei) [inline Makefile]: typo

**Review by atobiszei — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (rasapala) [reply Makefile]: Done

**Pull request merged**